script.js

```diff
@@ -0,0 +1,76 @@
+const loader = document.querySelector('.loader')
+const accessKey = 'DM7QQ7RJF8s-x7lfbfdF6_p-BikcD5s0CFQsUql_mvM'
```

Comment:
Don't have your access keys, secrets in a PR, store them separately in a file that is not included in git, and reference those files.